### PR TITLE
Readme updates

### LINKED ISSUE #2: Finish readme
Readme is largely unfinished at the moment. Any who would like to do this before I get to this is welcome to do so.
Should include:
- How to setup
- Information on Revolution like how it works e.g how it finds the request handler

### PR DESCRIPTION
**Pull request by littlemisssynth**
Resolves #2

### CODE REVIEW

**Review by tgpholly — CHANGES_REQUESTED** [review]
Maybe add more before requesting a review?

File: README.md [hunk 1]
```diff
@@ -14,15 +14,10 @@
 <p align="left">Modules are loaded in at runtime and are completely separate from each other. Each module has a prefix, for example the request handlers have the prefix “handle_console”. These prefixes determine the function of the modules and help the server to function.</p>
 <h2 align="left">How to use Revolution.</h2>
 <h3 align="left">From source</h3>
-<p align="left">git clone https://github.com/tgpethan/Revolution.git
-cd Revolution
-npm i
-node .
+<p align="left">git clone https://github.com/tgpethan/Revolution.git then, cd Revolution. run npm i, then run node .
```
Comment (tgpholly) [inline README.md]: Keep the instructions as original and place inside codeblocks to keep formatting correct

File: README.md [hunk 2]
```diff
@@ -14,15 +14,10 @@
...
 </p>
 <h3 align="left">From build</h3>
-<p align="left">Download latest build (https://github.com/tgpethan/Revolution/releases/latest)
-Unzip the zip
-Run revolution executable 
+<p align="left">Download latest build (https://github.com/tgpethan/Revolution/releases/latest) then Unzip the zip and Run revolution executable
```
Comment (tgpholly) [inline README.md]: Keep the instructions as original and place inside codeblocks to keep formatting correct

**Review by tgpholly — CHANGES_REQUESTED** [review]
It's mostly ok, just keep it the same as the mockup and place it in codeblocks.

**Review by tgpholly — COMMENTED** [review]
Weird It'll do. I'll add some final stuff and i'll approve it

**Comment by littlemisssynth** [comment]
oki

**Review by tgpholly — APPROVED** [review]

**Pull request merged**